src/o365/spo/commands/web/web-set.ts

```diff
@@ -44,6 +45,7 @@ class SpoWebSetCommand extends SpoCommand {
     telemetryProps.siteLogoUrl = typeof args.options.siteLogoUrl !== 'undefined';
     telemetryProps.title = typeof args.options.title !== 'undefined';
     telemetryProps.quickLaunchEnabled = typeof args.options.quickLaunchEnabled !== 'undefined';
+    telemetryProps.footerEnabled = typeof args.options.footerEnabled !== 'undefined';
```

Comment:
For `footerEnabled` it would be actually interesting to know if people turn footers on or off.